### DISCUSSION TITLE
32k mode processing recipes?

### DISCUSSION THREAD

**Discussion by o-smirnov** [caracal-pipeline/caracal#1401] [Q&A]
@paoloserra @Fil8, do you already have a set of standard recipes for 32k mode?

@rubyvanrooyen is getting into processing that with Caracal (and other people would like to jump in), so it would be nice to have some examples to work from.

There is also the question of bandpass calibration in "Bpoly" mode which is needed for zoom modes, but that's a separate discussion.

**Comment by paoloserra** [comment]
32k broadband continuum?
